libs/movie/src/lib/movie/form/summary/available-materials/available-materials.component.ts

```diff
@@ -24,4 +28,8 @@ export class SummaryAvailableMaterialsComponent implements OnInit {
     return this.movie.get('delivery').get('file').value;
   }
 
+  public async getDownloadUrl(file: StorageFile) {
+    const url = await this.mediaService.generateImgIxUrl(file, {});
+    window.open(url);
+  }
```

Comment:
You have access to file in the ts, no need to pass it to function.
Rename to downloadFile
```suggestion
  public async downloadFile() {
    const url = await this.mediaService.generateImgIxUrl(this.file, {});
    window.open(url);
  }
```

Reply:
and you don't need `, {}`